service/awsconfig/v10/templates/cloudformation/guest/instances.go

```diff
@@ -0,0 +1,37 @@
+package guest
+
+const Instances = `{{define "instance"}}
+  {{ .Instances.Master.Instance.ID }}:
```

Comment:
Using the new adapter structure here.

Reply:
I find this naming confusing. Instance ID's in AWS are the IDs provided by AWS in the format `i-098ed00a4e61a8e4f`.

This is the name of the Cloud Formation resource so `Master.Instance.ResourceName` makes more sense to me. I know we need the node ID to be added for updates.

Reply:
`ResourceName` it is now.